### DISCUSSION TITLE
revel m126be spinorama data posted on 23 June

### DISCUSSION THREAD

**Discussion by tyglander** [pierreaubert/spinorama#254] [General]
Erin seems to be posting reviews again (yay) and I noticed the Revel m126be is still using Revel's data instead of Erin's.  Here is a link to Erin's data: [Revel_m126be](https://www.erinsaudiocorner.com/loudspeakers/revel_m126be/)

**Comment by pierreaubert** [comment]
Hello,

I am waiting from Erin to send me the data.

Pierre

**Comment by tyglander** [comment]
Awesome!


Sent from Outlook<http://aka.ms/weboutlook>

________________________________
